Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -79,6 +81,10 @@ final class PostCampaignCheckoutViewController: UIViewController {
     self.viewModel.inputs.viewDidLoad()
   }
 
+  deinit {
+    self.sessionStartedObserver.doIfSome(NotificationCenter.default.removeObserver)
```

Comment:
Nifty, first time I've seen this in this codebase. Some kind of fancy RAC unsubscribe?